src/krux/pages/qr_view.py

```diff
@@ -134,16 +151,23 @@ def highlight_qr_region(self, code, region=(0, 0, 0, 0), zoom=False):
                             WHITE,
                         )
 
+    # TODO: add python typings
     def _region_legend(self, row, column):
+        """
```

Comment:
Don't add documentation without adding documentation plz 😆 This will make pylint to pass and nobody will see this anymore!